src/Application.cpp

```diff
@@ -1830,12 +1846,18 @@ void Application::handle(const SDL_KeyboardEvent* key)
 
   case KeyBinds::Action::kPauseToggle:
     if (_fsm.currentState() == Fsm::State::GamePaused)
+    {
+      _fsm.resumeGame();
+      RA_SetPaused(false);
```

Comment:
Because of the inversion mentioned above for Pause on Reset/Pause on Trigger, the FSM is no longer responsible for showing the overlay. Do so here.